applications/Shopbring.md

```diff
@@ -0,0 +1,320 @@
+* **Arbitrators.** Users who register as a arbitrator need to freeze a certain amount of native tokens. After becoming a arbitrator, they are mainly responsible for arbitrating purchase transaction disputes. Part of the fine can be obtained by completing the arbitration.
+* **Inspector**. To register as an inspector, a majority of the council is required. The inspector needs to run an off-chain service for verifying shopping receipts, such as verifying the authenticity of commodity invoices.
+* **Council.** Election of council members in accordance with Polkadot governance rules.
+* **Technical Committee.** Technical committees are established by the Council in accordance with Polkadot governance rules.
+* **Treasury.** Refer to the design of the Polkadot Treasury, the funds come from the token incentive model, transaction fees and tips, etc. As long as they contribute to the development of the community, anyone can apply for financial support.
+
+#### Credit Incentive Mechanism
+
+For the healthy development of Shopbring, we design a credit value growth model and native token incentive algorithm. The core meaning is as follows:
+
+* The behavior that is favorable to the development of Shopbring will increase the credit value, and the behavior that is unfavorable to the development of Shopbring will reduce the credit value.
+* Open and transparent credit data can help users choose trading partners.
+* In Polkadot ecosystem, it provides credit data reference for other parachains.
+
+In order to encourage the Shopbring users to increase the credit value, the system will determine the number of airdrop tokens according to the average credit rating of the overall users in each cycle. The higher the average credit value, the greater the total number of airdrop tokens. The number of airdrop tokens obtained by users = the number of single credit value airdrop tokens * the user credit value.
+
+#### Arbitrator
+
+Normal user registration arbitrators need to freeze a certain native token as an application condition. The `council` will review the applicant's information, and more than half of the members will vote to become an arbitrator.
+
+The arbitrator’s obligation is that when a common user has a dispute during the commissioned shopping, either party will appeal to the system. The arbitrator can choose the case to intervene in the dispute. During the waiting period, up to `M` arbitrators can sign up to participate in the dispute. After the waiting period ends, the system will randomly select `N`($N < M$, and `N` is an odd number) arbitrators to actually participate in the dispute. If there are less than `N` arbitrators actually participating, the `council` will participate in dispute arbitration. After the arbitration is completed, the `treasury` will pay wages to the participating arbitrators.
```

Comment:
In order to establish a good arbitration mechanism in the early stage of the network, the council will consider people with customer service experience to be arbitrators when reviewing applications. It is also considered that there are not many commissioned shopping disputes in the early stage of the network, so the wages of the arbitrator will be fixed and paid by the treasury. This can reduce the cost of both parties to the transaction and encourage more arbitrators to actively participate in dispute resolution.

When the network's credit incentive mechanism is perfected, all accounts with credit value have been rewarded with native tokens. At this time, both parties need to deposit a certain native tokens (also can be deposited by other account) for commissioned shopping. Once a dispute occurs during the process, the party who loses the arbitration will pay the arbitrators.